src/commands/export.ts

```diff
@@ -40,8 +40,19 @@ export const doExport = async (uri: Uri, document: TextDocument) => {
     const conf = await createConfigFile(document)
 
     try {
-      await marpCli('-c', conf.path, input.path, '-o', uri.fsPath)
-      env.openExternal(uri)
+      if (!marpConfiguration().get<boolean>('customConfig')) {
+        await marpCli('-c', conf.path, input.path, '-o', uri.fsPath)
+      } else {
+        /* If we  are using the custom config, set the working directory
+         * allow Marp CLI to search the document's workspace for a
+         * config file.
+         */
+        const currentDir = path.cwd()
+        process.chdir(path.dirname(document.uri.fsPath))
+        await marpCli(input.path, '-o', uri.fsPath)
```

Comment:
[Some options](https://github.com/marp-team/marp-cli#options) might have to override in CLI arguments for export the document as VS Code user expected. For example:

- Disable extra conversion modes such as [watch, server, and preview mode](https://github.com/marp-team/marp-cli#conversion-modes) (These bring looking like hang up conversion process in VS Code UI). `--no-server`, `--no-watch` and `--no-preview` will work.
- Prefer export type selected in the save dialog rather than specified in configuration file. (Require to pass specific flag: `--pdf`, `--pptx`, and `--image [png|jpeg]`)